eng/CodeChecks.ps1

```diff
@@ -79,6 +79,8 @@ try {
 
     Write-Host "Re-generating clients"
     Invoke-Block {
+        & dotnet msbuild $PSScriptRoot\service.proj /t:GenerateCode /p:Scope=$ServiceDirectory
```

Comment:
I'd use ServiceDirectory instead of Scope for the property. Scope is only a helper for the mgmt pipelines that will go away at some point.